nettacker/modules/scan/port.yaml

```diff
@@ -1028,7 +1028,7 @@ payloads:
           condition_type: or
           conditions:
             open_port:
-              regex: ""
+              regex: \d{{1,5}}
```

Comment:
this is wrong, it must be like

[0-9]{{1,5}}

\d accepts other alphabets digits.